### PR TITLE
:bug:fix: compass typo and css style

### PR DESCRIPTION
**Pull request by muclover**

### CODE REVIEW

**Comment by muclover** [comment]
#252

**Comment by muclover** [comment]
https://github.com/originjs/oss-evaluation-service/issues/242

**Comment by muclover** [comment]
#242

File: packages/shared-components/components/software-details/SoftwareDetails.vue [hunk 1]
```diff
@@ -1012,35 +1012,93 @@ const emits = defineEmits<{
           </div>
         </el-card>
         <el-card mb-6 w-626px>
-          <div mb-4 font-size-5 font-bold>代码提交频率</div>
-          <div mb-2 font-size-3>过去90天内平均每周代码提交次数。</div>
+          <div mb-2 font-size-5 font-bold>
+            代码提交频率
+            <el-tooltip
+              content="数据来源于Compass，关于指标的意义和结果可以查看 Compass 官网: https://oss-compass.org/"
```
Comment (flyfishzy) [inline packages/shared-components/components/software-details/SoftwareDetails.vue:1018]: 1. 提取成常量
2. 调整为：数据来源于开源指南针OSS Compass，关于指标的详细说明可以查看官网: https://oss-compass.org/zh/docs/docs/metrics-models/

**Pull request merged**